libs/ptg-ui-react-lib/src/lib/data-table/material-table/material-table.tsx

```diff
@@ -31,32 +31,35 @@ export interface PtgUiMaterialTableProps {
     filtering: boolean,
     paging:boolean,
     paginationPosition?:any
-    grouping?:boolean
+    grouping?:boolean,
+    detailForAccordion?: any
```

Comment:
Add type for it.

Reply:
I've added type for detailForAccordion as detailForAccordion as detailForAccordion?: (rowData: any) => React.ReactNode;